yt/fields/field_info_container.py

```diff
@@ -421,7 +421,7 @@ def add_output_field(self, name, sampling_type, **kwargs):
         kwargs.setdefault("ds", self.ds)
         self[name] = DerivedField(name, sampling_type, NullFunc, **kwargs)
 
-    def alias(self, alias_name, original_name, units=None, deprecate=None):
+    def alias(self, alias_name, original_name, units=None, deprecate:Optional[Tuple[str]]=None):
```

Comment:
I'm sorry I really didn't mean to suggest you actually used this !
this will break CI unless you also add
```python
from typing import Optional, Tuple
```
at the top !